benchmark/galsim/GalsimBenchmark.jl

```diff
@@ -164,6 +163,19 @@ function benchmark_comparison_data(params, header)
             exp(params[ids.c1[4, star_galaxy_index]]),
             params[ids.a[2]],
         ],
+        joint_infer_actual=Float64[
+            joint_infer_params[ids.u[1]],
+            joint_infer_params[ids.u[2]],
+            joint_infer_params[ids.e_axis],
+            canonical_angle(joint_infer_params) * 180 / pi,
+            joint_infer_params[ids.e_scale],
+            exp(joint_infer_params[ids.r1[star_galaxy_index]]),
+            exp(joint_infer_params[ids.c1[1, star_galaxy_index]]),
+            exp(joint_infer_params[ids.c1[2, star_galaxy_index]]),
+            exp(joint_infer_params[ids.c1[3, star_galaxy_index]]),
+            exp(joint_infer_params[ids.c1[4, star_galaxy_index]]),
+            joint_infer_params[ids.a[2]],            
```

Comment:
Looks like some duplicated code here. Would put the common code in a function called something like `vs_to_actual` and call it twice? Something like
```
single_infer_actual = vs_to_actual(params)
joint_infer_actual = vs_to_actual(joint_infer_params)
```
And maybe rename `params` to `single_ infer_params` for symmetry.